react/tic-tac-toe/src/app/screens/Game/components/Matches/index.js

```diff
@@ -0,0 +1,43 @@
+import React, { Component, Fragment } from 'react';
+
+import MatchesService from '../../../../../services/MatchesService';
+import { getWinnerClass } from '../../utils';
+
+const Spinner = require('react-spinkit');
+
+class Matches extends Component {
+  state = {
+    data: [],
+    loading: true
+  };
+
+  componentDidMount() {
+    const match = MatchesService.getMatches();
```

Comment:
Mmm `getMatches()` is plural and you set result in the const `match` that is in singular  👀